apps/server/src/modules/user-migration/service/user-migration.service.ts

```diff
@@ -0,0 +1,91 @@
+import { OauthConfig } from '@shared/domain';
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { EntityNotFoundError } from '@shared/common';
+import { SystemService } from '@src/modules/system/service/';
+import { PageContentDto } from './dto/page-content.dto';
+import { PageTypes } from '../interface/page-types.enum';
+import { SystemDto } from '../../system/service/dto/system.dto';
+
+@Injectable()
+export class UserMigrationService {
+	private readonly PROCESS_MIGRATION_BASE_URL: string = '/api/v3/migration/process';
```

Comment:
shouldn't this be a oauth endpoint like `/api/v3/sso/oauth/migration` or maybe `/api/v3/migration/oauth`, since we could have something like `api/v3/migration/email`?
Lets talk even more <3

Reply:
API_HOST is not reachable from outside, it gets passed trough the client trough an ingress route. about the route itself: lets keep it as a placeholder until we have a proper implementation